client/my-sites/site-settings/composing/jetpack-markdown.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const JetpackMarkdown = ( {
+	selectedSiteId,
+	translate
+} ) => {
+	return (
+		<div className="composing__module-settings">
+			<FormFieldset className="composing__form-fieldset site-settings__has-divider">
+				<div className="composing__info-link-container site-settings__info-link-container">
+					<InfoPopover position={ 'left' }>
+						<ExternalLink href={ 'https://jetpack.com/support/markdown/' } target="_blank">
+							{ translate( 'Learn more about Markdown.' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+				<JetpackModuleToggle
+					siteId={ selectedSiteId }
+					moduleSlug="markdown"
+					label={ translate( 'Write posts or pages in plain text Markdown syntax' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Write posts or pages in plain text Markdown syntax.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).